java/client/test/org/openqa/selenium/remote/RemoteWebDriverUnitTest.java

```diff
@@ -1065,7 +1065,12 @@ public void canHandleElementCssPropertyCommand() throws IOException {
         new CommandPayload(DriverCommand.GET_ELEMENT_VALUE_OF_CSS_PROPERTY,
                            ImmutableMap.of("id", element.getId(), "propertyName", "color")));
   }
-
+  
+  @Test
+  public void noArgConstuctorEmptyCapabilitiesTest() throws IOException {
+    RemoteWebDriver driver = new RemoteWebDriver();
```

Comment:
Perhaps override this and ensure that the capabilities are set properly, as per the bug? You can declare a new class inline:

```java
public void noArgConstructorCreatesEmptyCapabilities() {
  RemoteWebDriver driver = new RemoteWebDriver() {}; // anonymous subclass
  assertThat(driver.getCapabilities()).isEqualTo(new ImmutableCapabilities());
}
```